package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "client": "cd client && npm start",
-    "build": "cd client && npm run build",
+    "build": "cd client && npm run build build-storybook",
+    "storybook": "cd client && npm run storybook",
```

Comment:
The commit name should've been called Add proxy and build/storybook scripts ;)

Reply:
right! )